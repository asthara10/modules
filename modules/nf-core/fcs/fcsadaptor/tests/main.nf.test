nextflow_process {

    name "Test Process FCS_FCSADAPTOR"
    script "../main.nf"
    process "FCS_FCSADAPTOR"

    tag "modules"
    tag "modules_nfcore"
    tag "fcs"
    tag "fcs/fcsadaptor"

    test("test_fcs_fcsadaptor") {


        when {
            process {
                """
                
                input[0] = [
        [ id:'test', single_end:false ], // meta map
        file(params.test_data['bacteroides_fragilis']['genome']['genome_fna_gz'], checkIfExists: true)
    ]

                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out.versions).match("versions") }
                { assert snapshot(process.out).match() }
            )
        }

    }

}
