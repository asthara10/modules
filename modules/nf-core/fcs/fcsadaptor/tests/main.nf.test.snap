{
    "versions": {
        "content": [
            [
                
            ]
        ],
        "timestamp": "2023-11-24T14:32:49.013298"
    }
}